Source/WebInspectorUI/UserInterface/Base/Utilities.js

```diff
@@ -1255,6 +1279,8 @@ Object.defineProperty(String.prototype, "levenshteinDistance",
 {
     value(s)
```

Comment:
`String.prototype.levenshteinDistance` actually doesnt appear to be used anywhere so we should just remove it instead